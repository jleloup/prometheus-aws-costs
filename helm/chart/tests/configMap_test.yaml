---
suite: Test ConfigMap
templates:
  - config.yaml
tests:
  - it: creates ConfigMap
    set:
      config:
        FAKE_VARIABLE: "test"
    asserts:
      - matchSnapshot: {}
