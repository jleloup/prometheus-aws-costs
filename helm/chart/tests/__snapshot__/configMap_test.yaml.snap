creates ConfigMap:
  1: |
    apiVersion: v1
    data:
      FAKE_VARIABLE: test
      OTEL_EXPORTER_OTLP_ENDPOINT: ""
      SERVICE_NAME: prometheus-aws-costs
      SERVICE_VERSION: "0"
      TELEMETRY_ENABLED: "true"
    kind: ConfigMap
    metadata:
      annotations: null
      labels:
        app.kubernetes.io/instance: RELEASE-NAME-prometheus-aws-costs
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: prometheus-aws-costs
        app.kubernetes.io/version: 0.1.0
        helm.sh/chart: prometheus-aws-costs-0.1.0
      name: prometheus-aws-costs
